CMakeLists.txt

```diff
@@ -57,6 +57,14 @@ if(NOT OT_PLATFORM IN_LIST OT_EXAMPLE_PLATFORMS)
     message(FATAL_ERROR "Platform unknown: ${OT_PLATFORM}")
 endif()
 
+# OT_CONFIG allows users to specify the path to OpenThread project core
+# config header file. The default value of this parameter is "none".
+# When not specified by user (value is "none"), a platform cmake file may
+# choose to change this variable to provide its own core config header
+# file instead.
+
+set(OT_CONFIG "none" CACHE STRING "OpenThread project-specific config header file chosen by user at configure time")
```

Comment:
Personally I think I prefer "" to be the default value for variables in the case that they have not been selected. An empty string is treated as FALSE for if() statements, so slightly more intuitive.

Reply:
Thanks! Great suggestion. Will change it.